spec/lib/jobs/install-os-job-spec.js

```diff
@@ -278,4 +278,82 @@ describe('Install OS Job', function () {
             });
         });
     });
+
+    describe('test _validateOptions', function() {
+
+        it('should throw error when username is not given ', function() {
+            job.options.users = [{password:'12345', uid:600, sshKey:''}];
+            return expect(job._validateOptions.bind(job,{}))
+                   .to.throw('username is required');
+        });
+
+        it('should throw error when username is not valid', function() {
+            job.options.users = [
+                {"name":"WangW25", "uid":1066, "password":"12345", "sshKey":"123456" },
+                {"name": "", "uid":1069, "password":"12345", "sshKey":"123456"}];
+            return expect(job._validateOptions.bind(job,{}))
+                   .to.throw('username is required');
+        });
+
+        it('should throw error when username is not a string ', function() {
+            job.options.users = [{name:100, password:'12345', uid:600, sshKey:''}];
+            return expect(job._validateOptions.bind(job,{}))
+                   .to.throw('username must be a string');
+        });
+
+        it('should throw error when username is not valid', function() {
```

Comment:
This description is same with line#290, it's better to change to a different one.